besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -592,14 +592,6 @@ static class PrivacyOptionGroup {
       defaultValue = "localhost,127.0.0.1")
   private final JsonRPCAllowlistHostsProperty hostsAllowlist = new JsonRPCAllowlistHostsProperty();
 
-  @Option(
-      names = {"--host-whitelist"},
-      hidden = true,
-      paramLabel = "<hostname>[,<hostname>...]... or * or all",
-      description =
-          "Deprecated in favor of --host-allowlist. Comma separated list of hostnames to allow for RPC access, or * to accept any host (default: ${DEFAULT-VALUE})")
-  private final JsonRPCAllowlistHostsProperty hostsWhitelist = new JsonRPCAllowlistHostsProperty();
```

Comment:
There is also some code in the configure method of BesuCommand that uses the hostsWhitelist variable that needs to be removed as well

Reply:
Thanks for the review! You mean this part further down in the same file, of another one?

```
    // hostsWhitelist is a hidden option. If it is specified, add the list to hostAllowlist
    if (!hostsWhitelist.isEmpty()) {
      // if allowlist == default values, remove the default values
      if (hostsAllowlist.size() == 2
          && hostsAllowlist.containsAll(List.of("localhost", "127.0.0.1"))) {
        hostsAllowlist.removeAll(List.of("localhost", "127.0.0.1"));
      }
      hostsAllowlist.addAll(hostsWhitelist);
    }
```

Reply:
@macfarla Thanks, unfortunately we were still not able to run the repo locally... Could you guide us on which piece of code to remove?

Reply:
@JanetMo feel free to post in [discord](https://discord.com/channels/905194001349627914/938504958909747250) if you're having trouble compiling or building locally, plenty of folks there that can help you out

Reply:
@macfarla Thanks, appreciate the support. I retried but still wasn't able to build it locally, is there a way to unassign from the issue? I think I'm not up for it, and probably holding you back on with this one.

Reply:
ok yes I can unassign you @JanetMo - you're welcome back any time - but being able to compile the code is a must :)